cmake/modules/gluecodium/gluecodium/Generate.cmake

```diff
@@ -170,7 +173,7 @@ function(apigen_generate)
       PRIVATE ${apigen_generate_CPP_EXPORT}_INTERNAL)
   endif()
 
-  message("Configured `Gluecodium" ${APIGEN_GLUECODIUM_ARGS})
+  message("Configured `Gluecodium")
```

Comment:
What's the extra character before "Gluecodium"?

Reply:
I don't know, it was done by @Dschoordsch

Reply:
Suspect it's a typo. You probably can check with him in dependent from this patch.